scss/list-book.scss

```diff
@@ -0,0 +1,37 @@
+@import 'colors';
```

Comment:
No need to import this in here. You can import it in the `application.scss` and all stylesheets can use it 😄